edx_when/models.py

```diff
@@ -34,12 +40,21 @@ def __str__(self):
         # TODO: return a string appropriate for the data fields
         return '<DatePolicy, ID: {}>'.format(self.id)
 
-    @property
-    def actual_date(self):
+    def actual_date(self, schedule=None):
```

Comment:
Breaking change, yah? Shall we bump the major version and such then? I see we're still on `0`.

Reply:
Yeah, good call, we should bump the version. I think it would be reasonable to go to 1.0 after we finish the work that we're doing as part of this AA work.